jet/cdc/src/main/java/com/hazelcast/samples/jet/cdc/CdcRealTimeAnalysisDemo.java

```diff
@@ -0,0 +1,142 @@
+package com.hazelcast.samples.jet.cdc;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.JetService;
+import com.hazelcast.jet.Traverser;
+import com.hazelcast.jet.cdc.ChangeRecord;
+import com.hazelcast.jet.cdc.Operation;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+
+import static com.hazelcast.jet.Traversers.singleton;
+import static com.hazelcast.jet.Traversers.traverseItems;
+import static com.hazelcast.jet.cdc.Operation.DELETE;
+import static com.hazelcast.jet.cdc.Operation.INSERT;
+import static com.hazelcast.jet.cdc.postgres.PostgresCdcSources.postgres;
+import static com.hazelcast.jet.datamodel.Tuple2.tuple2;
+import static com.hazelcast.samples.jet.cdc.CdcRealTimeAnalysisDemo.ReportEvent.eventFor;
+import static com.hazelcast.samples.jet.cdc.CdcRealTimeAnalysisDemo.ReportEvent.orderEvent;
+
+/**
+ * Demonstrates a simple reporting pipeline, which will be attached to existing system without the need of any change
+ * in the upstream system and will update some analytics based on actions done by customers.
+ *
+ * To run this example you have to have PostgreSQL database set up. You can quickly do this, by running in your terminal:
+ * <pre>
+ *     docker run -d --rm --name cdc-postgres -p 5432:5432 \
+ *      -e POSTGRES_DB=postgres -e POSTGRES_USER=postgres \
+ *      -e POSTGRES_PASSWORD=cdcpwd1337 debezium/example-postgres:1.9.3.Final
+ * </pre>
+ *
```

Comment:
I was a bit surprised that I couldn't find the place where we populate the DB. Maybe it would be worth mentioning that `debezium/example-postgres` is filled with sample data or even adding a code that changes DB data to see that the integration works in the real time.